commit 768ab13cf4cbc76ca4e43f6476ed139c396896c7
Author: Kevin Ottens <ervin@kde.org>
Date:   Tue Mar 6 14:13:54 2018 +0100

    Fix linking of the cuke-steps
    
    Turns out that cucumber-cpp via its boost dependency is pulling also the
    pthreads. In case of a string linker this requires to be linked against
    as well.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a6191f54..0de24b51 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,6 +15,7 @@ include(ECMPoQmTools)
 
 find_package(Qt5 ${QT_REQUIRED_VERSION} CONFIG REQUIRED Core Gui Widgets Qml Test)
 find_package(Boost REQUIRED)
+find_package(Threads REQUIRED)
 macro(assert_min_ver version)
     set(error_msg "${CMAKE_CXX_COMPILER} ${CMAKE_CXX_COMPILER_VERSION} not supported")
     if("${CMAKE_CXX_COMPILER_VERSION}" VERSION_LESS "${version}")
diff --git a/tests/features/renku/features/step_definitions/CMakeLists.txt b/tests/features/renku/features/step_definitions/CMakeLists.txt
index 03096058..d272e6af 100644
--- a/tests/features/renku/features/step_definitions/CMakeLists.txt
+++ b/tests/features/renku/features/step_definitions/CMakeLists.txt
@@ -13,6 +13,7 @@ set(steps_SRCS
 add_executable(renku-cuke-steps ${steps_SRCS})
 target_link_libraries(renku-cuke-steps
     cucumber-cpp
+    Threads::Threads
     ${Boost_REGEX_LIBRARY}
     ${Boost_SYSTEM_LIBRARY}
     ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY}
diff --git a/tests/features/zanshin/features/step_definitions/CMakeLists.txt b/tests/features/zanshin/features/step_definitions/CMakeLists.txt
index 72540c15..ea270fbe 100644
--- a/tests/features/zanshin/features/step_definitions/CMakeLists.txt
+++ b/tests/features/zanshin/features/step_definitions/CMakeLists.txt
@@ -13,6 +13,7 @@ set(steps_SRCS
 add_executable(zanshin-cuke-steps ${steps_SRCS})
 target_link_libraries(zanshin-cuke-steps
     cucumber-cpp
+    Threads::Threads
     ${Boost_REGEX_LIBRARY}
     ${Boost_SYSTEM_LIBRARY}
     ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY}
